imod/mf6/gwfgwf.py

```diff
@@ -1,23 +1,24 @@
-from typing import Dict, Optional, Tuple
+from typing import Optional
 
 import cftime
 import numpy as np
 import xarray as xr
 
 from imod.mf6.auxiliary_variables import add_periodic_auxiliary_variable
+from imod.mf6.exchangebase import ExchangeBase, ExchangeType
 from imod.mf6.package import Package
 from imod.typing import GridDataArray
 
 
-class GWFGWF(Package):
+class GWFGWF(ExchangeBase):
     """
     This package is for writing an exchange file, used for splitting up a model
     into different submodels (that can be solved in parallel). It (usually)
     is not instantiated by users, but created by the "split" method of the
     simulation class."""
 
-    _keyword_map: Dict[str, str] = {}
     _auxiliary_data = {"auxiliary_data": "variable"}
+    _exchange_type = ExchangeType.GWFGWF
```

Comment:
isn't this a bit double? we set package_id below to gwfgwf. Can the _exchange_type vary independently from the package_id? Would a mapping from package_id to exchange type make sense?

Reply:
You're right. This isn't really  necesarry. I removed it in favor of a mapping dictionary